nft/service.go

```diff
@@ -257,6 +260,12 @@ func (s *service) minterJob(ctx context.Context, tokenID TokenID, model document
 		if req.UseGeneric { // TODO Remove once we have finalized the generic NFT work
```

Comment:
We can start thinking on dropping this branch, and default always to generic. I think we can do it after we are done with Amber release.

Reply:
yeah